fullscreen.bs

```diff
@@ -1,3 +1,60 @@
+<pre class=metadata>
+Title: Fullscreen API Standard
+Group: WHATWG
+H1: Fullscreen API
+Shortname: fullscreen
+Status: LS
+No Editor: true
+Abstract: The Fullscreen API standard defines an API for elements to display themselves fullscreen.
+Logo: https://resources.whatwg.org/logo-fullscreen.svg
+Boilerplate: omit feedback-header, omit conformance, omit index, omit idl-index
+!Participate: <a href=https://github.com/whatwg/fullscreen>GitHub whatwg/fullscreen</a> (<a href=https://github.com/whatwg/fullscreen/issues/new>file an issue</a>, <a href=https://github.com/whatwg/fullscreen/issues>open issues</a>)
+!Participate: <a href=https://wiki.whatwg.org/wiki/IRC>IRC: #whatwg on Freenode</a>
+!Commits: <a href=https://github.com/whatwg/fullscreen/commits>GitHub whatwg/fullscreen/commits</a>
+!Commits: <a href=https://twitter.com/fullscreenapi>@fullscreenapi</a>
+</pre>
+
+<script src=https://resources.whatwg.org/file-issue.js async></script>
+<script id=head src=https://resources.whatwg.org/dfn.js defer></script>
+
+<pre class=anchors>
+urlPrefix: https://dom.spec.whatwg.org/
+    type: dfn
+        text: concept document
+        text: concept element
+        text: concept element namespace
+        text: concept event fire
+        text: concept node document
+        text: concept node remove ext
+        text: concept tree ancestor
+        text: concept tree inclusive ancestor
+        text: concept tree inclusive descendant
+        text: concept tree order
+        text: concept tree root
+    type: interface
+        text: Document
+urlPrefix: https://html.spec.whatwg.org/multipage/
```

Comment:
It looks like "browsing context" can be dropped from this list, but Bikeshed doesn't know about "ancestor browsing context", "browsing context container" or "unloading document cleanup steps".